ucfcs/GrowthPlus
# Level234 styling & clean up

- Finished up the fragments for levels 2, 3, 4
- Added translations and styling for different lessons and flashcards
- Added clearer images for some lessons
- Adjusted colors for lesson fragments 

TEST 
- Go through all the lesson tiles for levels 2,3, and 4
- Make sure nothing breaks and all the text is visible
- Check that translations are working properly 
- Complete flashcards and make sure that they work correctly
